### PR TITLE
DEV 2863 standardize structure

### PR DESCRIPTION
**Pull request by qqiao2024**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -5,6 +5,7 @@ on:
       - develop
       - release/**
       - hotfix/**
+      - feat/dev-2863-standardize-structure
```
Comment (mpsolano) [inline .github/workflows/ci.yaml]: Should this be removed before merge?

**Review by mpsolano — APPROVED** [review]
Looks good!

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (qqiao2024) [reply .github/workflows/ci.yaml]: removed

File: setup.cfg [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+maintainer_email = gdc_dev_questions-aaaaae2lhsbell56tlvh3upgoq@cdis.slack.com
+license = Apache Software License 2.0
+license_files = LICENSE
+url = https://github.com/NCI-GDC/gdc-client
+keywords = gdc, client, download, upload
+classifiers =
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+		Programming Language :: Python :: 3.9
```
Comment (kulgan) [inline setup.cfg]:
```suggestion
	Programming Language :: Python :: 3.9
```

File: setup.cfg [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+[options.extras_require]
+dev =
+    click>= 8
+    flask~=1.0.2
+    pytest~=4.6.2
+    pytest-cov
+    requests-mock~=1.5.2
+    moto~=1.3.14
+    python-jose<3.3.0       # lower the requirements for cryptography, so python3.5 could work
```
Comment (kulgan) [inline setup.cfg:54]: project no longer supports python3.5, we might be able to remove the comment at least and possibly remove the pins (pytest pins should be bumped no matter what though)

File: pyproject.toml [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+[tool.coverage.html]
+title = "gdc-client coverage report"
+directory = "docs/htmlcov"
+show_contexts = true
+
+[tool.coverage.run]
+branch = true
+context = "unit tests"
+source = ["gdc-client"]
```
Comment (kulgan) [inline pyproject.toml]:
```suggestion
source = ["gdc_client"]
```

File: pyproject.toml [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+[tool.isort]
+extend_skip = [".md", ".json"]
+known_first_party = ["gdc-client"]
+py_version = 37
```
Comment (kulgan) [inline pyproject.toml]: ?
```suggestion
known_first_party = ["gdc_client"]
py_version = 38
```

File: setup.cfg [hunk 6]
```diff
@@ -0,0 +1,61 @@
+[metadata]
+name = gdc_client
```
Comment (kulgan) [inline setup.cfg]: To reduce confusion, make this consistent with pyproject.toml
```suggestion
name = gdc-client
```

File: pyproject.toml [hunk 7]
```diff
@@ -0,0 +1,42 @@
+[build-system]
```
Comment (kulgan) [inline pyproject.toml:1]: I would add the pytest config here as well and enable printing coverage to terminal either in the pyproject.toml or in the tox file

File: src/gdc_client/cli.py
Comment (kulgan) [inline src/gdc_client/cli.py:1]: 👍🏾

File: setup.cfg (hunk 3 above)
Comment (qqiao2024) [reply setup.cfg:54]: tried to bump up pins, run into lots of troubles.

**Review by kulgan — APPROVED** [review]

**Pull request merged**